examples/asr/conf/fastconformer/fast-conformer-long_ctc_bpe.yaml

```diff
@@ -0,0 +1,203 @@
+# It contains the default values for training a Fast Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+
+# You may find more info about Fast Conformer here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#fast-conformer
+
+# This version uses Longformer-style attention in order to handle longer audio
```

Comment:
Let's move this part about the Longformer to the beginning. It would be also nice to give some hints why and when users may need to use this training. Maybe discuss about the global token feature?

Reply:
Moved part higher. Added mention of which parameters are changed, the parameters are pretty self-explanatory. The docs explains when training is needed